app/scripts/AxisPixi.js

```diff
@@ -261,7 +262,8 @@ class AxisPixi {
     if (this.track.getTheme() === THEME_DARK) stroke = '#cccccc';
 
     const line = document.createElement('path');
-    line.setAttribute('id', 'tick-mark');
+    const lineAxisSVGUniqueId = Math.random().toString(36).substring(7);
```

Comment:
We typically use slugid for random identifiers.

See https://github.com/higlass/higlass/blob/develop/app/scripts/TrackRenderer.jsx#L272.

Reply:
Resolved